sdk/documentintelligence/azure-ai-documentintelligence/samples/sample_analyze_custom_documents.py

```diff
@@ -30,42 +30,8 @@
 import os
 from collections import Counter
 
-def print_table(header_names, table_data):
-    """Print a two-dimensional array like a table.
-
-    Based on provided column header names and two two-dimensional array data, print the strings like table.
-
-    Args:
-        header_names: An array of string, it's the column header names.  e.g. ["name", "gender", "age"]
-        table_data: A two-dimensional array, they're the table data.  e.g. [["Mike", "M", 25], ["John", "M", 19], ["Lily", "F", 23]]
-    Return: None
-        It's will print the string like table in output window. e.g.
-         Name    Gender    Age
-         Mike    M         25
-         John    M         19
-         Lily    F         23
-    """
-    max_len_list = []
-    for i in range(len(header_names)):
-        col_values = list(map(lambda row: len(str(row[i])), table_data))
-        col_values.append(len(str(header_names[i])))
-        max_len_list.append(max(col_values))
-
-    row_format_str = "".join(map(lambda len: f"{{:<{len + 4}}}", max_len_list))
-
-    print(row_format_str.format(*header_names))
-    for row in table_data:
-        print(row_format_str.format(*row))
-
 
 def analyze_custom_documents(custom_model_id):
-    # For the Form_1.jpg, it should be the test file under the traning dataset which storage at the Azure Blob Storage path
-    # combined by DOCUMENTINTELLIGENCE_STORAGE_CONTAINER_SAS_URL and DOCUMENTINTELLIGENCE_STORAGE_PREFIX, 
-    # or it can also be a test file with the format similar to the training dataset. 
-    # Put it here locally just for presenting documents visually in sample.
-
-    # Before analyzing a custom document, should upload the related training dataset into Azure Storage Blob and
-    # train a model. For more information, access https://aka.ms/build-a-custom-model please.
```

Comment:
Nit: Doesnt seem like we should remove this comment

Reply:
Added it back